--- package/pkgconf/pkgconf.mk	2022-10-10 15:02:34.832354494 +0000
+++ package/pkgconf/pkgconf.mk.orig	2022-10-10 15:02:08.280354389 +0000
@@ -5,7 +5,7 @@
 ################################################################################
 
 PKGCONF_VERSION = 0.8.9
-PKGCONF_SITE = http://rabbit.dereferenced.org/~nenolod/distfiles
+PKGCONF_SITE = http://mirrors.mit.edu/osmc/osmc/download/buildroot/2014.05
 PKGCONF_SOURCE = pkgconf-$(PKGCONF_VERSION).tar.bz2
 PKGCONF_LICENSE = pkgconf license
 PKGCONF_LICENSE_FILES = COPYING
